mmv1/products/bigquerybiglake/Database.yaml

```diff
@@ -0,0 +1,110 @@
+  api: "https://cloud.google.com/bigquery/docs/reference/biglake/rest/v1/projects.locations.catalogs.databases"
+base_url: "projects/{{project}}/locations/{{location}}/catalogs/{{catalog_id}}/databases"
+self_link: "projects/{{project}}/locations/{{location}}/catalogs/{{catalog_id}}/databases/{{name}}"
+create_url: "projects/{{project}}/locations/{{location}}/catalogs/{{catalog_id}}/databases?databaseId={{name}}"
+update_mask: true
+update_verb: :PATCH
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "bigquery_biglake_database"
+    primary_resource_id: "database"
+    vars:
+      name: "my_database"
+      catalog_id: "my_catalog"
+      bucket_id: "my_bucket"
+parameters:
+  - !ruby/object:Api::Type::String
+    name: "location"
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The geographic location where the Database should reside.
+  - !ruby/object:Api::Type::String
+    name: "name"
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The name of the database.
+  - !ruby/object:Api::Type::String
+    name: "catalog_id"
+    url_param_only: true
+    description: |
+      The name of the parent catalog.
+properties:
+  - !ruby/object:Api::Type::String
+    name: "createTime"
+    description: |
+      Output only. The creation time of the database. A timestamp in RFC3339
+      UTC "Zulu" format, with nanosecond resolution and up to nine fractional
+      digits. Examples: "2014-10-02T15:01:23Z" and
+      "2014-10-02T15:01:23.045123456Z".
+    output: true
+  - !ruby/object:Api::Type::String
+    name: "updateTime"
+    description: |
+      Output only. The last modification time of the database. A timestamp in
+      RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine
+      fractional digits. Examples: "2014-10-02T15:01:23Z" and
+      "2014-10-02T15:01:23.045123456Z".
+    output: true
+  - !ruby/object:Api::Type::String
+    name: "deleteTime"
+    description: |
+      Output only. The deletion time of the database. Only set after the
+      database is deleted. A timestamp in RFC3339 UTC "Zulu" format, with
+      nanosecond resolution and up to nine fractional digits. Examples:
+      "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+    output: true
+  - !ruby/object:Api::Type::String
+    name: "expireTime"
+    description: |
+      Output only. The time when this database is considered expired. Only set
+      after the database is deleted. A timestamp in RFC3339 UTC "Zulu" format,
+      with nanosecond resolution and up to nine fractional digits. Examples:
+      "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+    output: true
+
+  - !ruby/object:Api::Type::Enum
+    name: "type"
+    description: |
+      The database type.
+    values:
+      - :HIVE
+  - !ruby/object:Api::Type::NestedObject
+    name: "hiveOptions"
```

Comment:
Should this be required for now? It can be removed later if more types are added, but I'd guess it doesn't make sense to provision a hive DB without specifying the options

Reply:
done.